### PR TITLE
Implement web2018

### PR DESCRIPTION
**Pull request by obieler**
Apache config to use include_module already done:
-- In `/var/www/vhosts/archiveweb.epfl.ch/private/archiveweb.conf`:
```
<Directory ...>
    ....

    Options Includes
</Directory>
# For includes web2018 in common folder (for archiveweb homepage)
<IfModule mime_module>
     AddType text/html .html
     AddOutputFilter INCLUDES .html
</IfModule>
```

**Todo before deploy:**
- Add "kis" followin cron job:
`0 2 * * * sudo -u vftp wget -P /var/www/vhosts/archiveweb.epfl.ch/htdocs/common/includes-web2018/ -N -nd -r -l 1 https://web2018.epfl.ch/download-me.html`

### CODE REVIEW

**Comment by obieler** [comment]
good William, waiting for cookie-consent

File: README.md [hunk 1]
```diff
@@ -1,31 +1,24 @@
 Resume
 ------
 
-Tool for display the archived websites on the homepage of archiveweb.epfl.ch.
+- Homepage of archiveweb.epfl.ch that displays the liste of archived websites.
```
Comment (williambelle) [inline README.md]: Typo: list

File: bin/docker-dev-init.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+wget -P /usr/local/apache2/htdocs/common/includes-web2018/ -N -nd -r -l 1 https://web2018.epfl.ch/download-me.html
```
Comment (williambelle) [inline bin/docker-dev-init.sh]: shebang missing from first line

File: homepage/index.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  <header class="header header-light">
+    <!--#include virtual="/common/includes-web2018/drawer-en.html" -->
+
+    <div class="header-light-content">
+      <!--#include virtual="/common/includes-web2018/logo.html" -->
+
+      <h1>Web Archives</h1>
+      <ul aria-hidden="true" class="nav-header d-none d-xl-flex"></ul>
+
+      <!--#include virtual="/common/includes-web2018/search-fr.html" -->
```
Comment (williambelle) [inline homepage/index.html]: Should be `search-en.html`

**Comment by williambelle** [comment]
We need to add copyright on new files.

File: README.md (hunk 1 above)
Comment (obieler) [reply README.md]: :+1:

File: bin/docker-dev-init.sh (hunk 2 above)
Comment (obieler) [reply bin/docker-dev-init.sh]: :+1:

File: homepage/index.html (hunk 3 above)
Comment (obieler) [reply homepage/index.html]: :+1:

**Pull request merged**